plugins/wp-songs-library/includes/class-wp-songs-library-cli.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * Class Wp_Songs_Library_Cli
+ */
+class Wp_Songs_Library_Cli extends WP_CLI_Command {
+	public function __construct() {
+		WP_CLI::add_command( 'wsl', $this );
+	}
+
+	public function generate_sample_posts( $args, $args_assc ) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this open curly brace to the beginning of the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://54.213.144.90:9000/sonar/coding_rules#rule_key=php%3AS1808)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this parameter "$args_assc" to match the regular expression ^[a-z][a-zA-Z0-9]*$. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://54.213.144.90:9000/sonar/coding_rules#rule_key=php%3AS117)